ragnarok_packets/src/lib.rs

```diff
@@ -1793,6 +1800,21 @@ pub struct ObjectiveDetails1 {
     pub mob_name: String,
 }
 
+#[derive(Debug, Clone, Packet, ServerPacket, MapServer)]
+#[cfg_attr(feature = "interface", derive(korangar_interface::elements::PrototypeElement))]
+#[header(0x008A)]
+pub struct NotificationPacket {
+    pub entity_id: EntityId,
+    pub target_id: u32,
+    pub server_tick: u32,
+    pub src_speed: u32,
+    pub dmg_speed: u32,
+    pub damage: i16,
+    pub div: u16,
+    pub p_type: u8,
+    pub damage2: i16,
```

Comment:
We try to avoid abbreviation in the code base, so `src_speed` should be `source_speed` etc.

Also wondering what `p_type` and `damage2` are in this context.

Reply:
I will change to remove abbreviation